geometry_msgs/Vector3 eta_1
geometry_msgs/Vector3 eta_2
geometry_msgs/Vector3 ni_1
geometry_msgs/Vector3 ni_2
float64 prova